### PR TITLE
[mass_mailing_distribution_list][IMP]

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
Use a domain to define if merge wizard is into dl or newsletter context

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
:+1:

**Comment by Olivier-LAURENT** [comment]
:+1:

File: mass_mailing_distribution_list/wizard/merge_distribution_list.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Authors: Nemry Jonathan
+#    Copyright (c) 2014 Acsone SA/NV (http://www.acsone.eu)
+#
```
Comment (lmignon) [inline mass_mailing_distribution_list/wizard/merge_distribution_list.py:5]: Use a shorter licence description

File: mass_mailing_distribution_list/__openerp__.py [hunk 2]
```diff
@@ -52,6 +52,7 @@
     'data': [
         'views/mass_mailing.xml',
         'views/distribution_list_view.xml',
+        'wizard/merge_distribution_list_view.xml',
     ],
```
Comment (lmignon) [inline mass_mailing_distribution_list/__openerp__.py:57]: additionnel à licence key

**Comment by Cedric-Pigeon** [comment]
:+1:  code review but Travis always fails...

**Comment by adrienpeiffer** [comment]
@Cedric-Pigeon Travis is green

**Comment by Cedric-Pigeon** [comment]
@adrienpeiffer Indeed, changed since my last view => :+1:

**Pull request merged**